tests/test_cli.py

```diff
@@ -6,11 +6,9 @@
 
 import pytest
 
+from saltyrtc.server import Server
 from saltyrtc.server import __version__ as _version
-from saltyrtc.server import (
-    Server,
-    util,
-)
+from saltyrtc.server import util
```

Comment:
Mhh, I don't see this as more readable. Is there a flag to turn this off in isort?

Reply:
I'm not sure, I just applied the tool to the file because the tests failed. That would be something for a separate PR.

I agree that it looks ugly.